_posts/en/newsletters/2021-12-15-newsletter.md

```diff
@@ -0,0 +1,117 @@
+    writing.
+
+- **Preparing LN for PTLCs:** Bastien Teinturier started a
+  [thread][teinturier post] on the Lightning-Dev mailing list about
+  making the minimal [set of changes][ln docs 16] to the LN
+  communication protocol necessary to allow nodes to begin upgrading to
+  using [PTLCs][topic ptlc].  PTLCs are more private than the
+  currently-used [HTLCs][topic htlc] and use less block space.
+
+    Teinturier is trying to produce a set of changes that can be made at
+    the same time the as the proposed `option_simplified_update`
+    [protocol change][bolts #867] (see [Newsletter #120][news120
+    opt_simp_update]).  A secondary goal is trying to make the
+    communication protocol compatible with the fast-forward's based PTLC
+    protocol described in [Newsletter #152][news152 ff].  This will allow
+    nodes to upgrade in stages, first to `option_simplified_update` with
+    HTLCs, then to PTLCs, then to fast forwards.
+
+## FIXME:bitschmidty special sections
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding none yet this week
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23716][] adds a native Python implementation of
+  RIPEMD-160 to Bitcoin Core's test code.  This allows Bitcoin Core to
+  no longer use the RIPMD-160 function from Python `hashlib` library
+  which wraps the OpenSSL implementation of RIPEMD-160.  Newer versions
+  of OpenSSL no longer provide RIPEMD-160 support by default, requiring
+  it to be enabled separately.
+
+- [Bitcoin Core #20295][] rpc: getblockfrompeer FIXME:Xekyo
+
+- [Bitcoin Core #14707][] updates several RPCs to also include bitcoins
+  received in miner coinbase transactions.  A new
+  `include_immature_coinbase` option to the RPCs allows toggling
+  the inclusion of coinbase transactions before they are mature (have
+  100 confirmations, the earliest they can be spent according to
+  consensus rules).
+
+- [Bitcoin Core #23486][] updates the `decodescript` RPC to only return
+  the P2SH or P2WSH addresses for a script if that script can be used,
+  respectively, with P2SH or P2WSH.
+
+- [BOLTs #940][] deprecates the announcement and parsing of Tor
+  v2 onions in `node_announcements`. [Rust-Lightning #1204][], also
+  merged this week, updates that implementation to follow this updated
+  specification.
+
```

Comment:
I added https://github.com/lightning/bolts/pull/918 in a PR in @harding 's repo at https://github.com/harding/bitcoinops.github.io/pull/23 The PR is about dropping the requirement to ratelimit ping and pong messages which ensures higher quality of service for LSPs

Reply:
I did think this was minor, but I like the description of how it benefits LSPs.  Merged.  Thanks!